week3/homework/src/write.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+
+function write(req, res, toBeWritten) {
+  return new Promise((resolve, reject) => {
```

Comment:
nice that you promisified fs.writeFile 👍

Reply:
thanks man